hack/verify-deps.sh

```diff
@@ -2,11 +2,16 @@
 set -euo pipefail
 
 function print_failure {
-  echo "There appear to be uncommitted changes to go.mod. Please run 'go tidy' and 'go vendor'."
+  echo "There are unexpected changes to the vendor tree following `go mod vendor` and `go mod tidy`. Please"
+  echo "run these commands locally and double-check the Git repository for unexpected changes which may"
+  echo "need to be committed."
   exit 1
 }
 
-go mod vendor
-go mod tidy
+if [ "${OPENSHIFT_CI:-false}" = true ]; then
```

Comment:
So now the entire verification step is skipped outside of CI context?  I suppose that is fine.  It might be better to add a separate make target for CI-specific verification, but maybe that would be more effort than it would be worth.

Reply:
Yes. For now just run the script directly and set the env var.